com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxF.shader

```diff
@@ -120,7 +120,8 @@ Shader "HDRP/AxF"
 
 
 //      [ToggleUI] _EnableFogOnTransparent("Enable Fog", Float) = 1.0
-//      [ToggleUI] _EnableBlendModePreserveSpecularLighting("Enable Blend Mode Preserve Specular Lighting", Float) = 1.0
+        // We never set this as it is always 1, but its definition is needed when performing blending in Material.hlsl
+        [HideInInspector][ToggleUI] _EnableBlendModePreserveSpecularLighting("Enable Blend Mode Preserve Specular Lighting", Float) = 1.0
```

Comment:
it is not true (at least for now as _BLENDMODE_PRESERVE_SPECULAR_LIGHTING is still a keyword)

Reply:
I am referencing the property not keyword here, beforehand the property was not used, here I instead just define it (the keyword is still used as it should)

Reply:
In any case it is actually not true anymore (is not needed anymore after some subsequent changes I made) so I will revert this bit tomorrow.